src/components/SearchableDropdownList.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import SearchField from './SearchField';
+import SearchableDropdownListItem from './SearchableDropdownListItem';
+
+const SearchableDropdownList = ({ places, handleListItemClick }) => {
+  const [filteredSearches, setFilteredSearches] = useState([]);
```

Comment:
it;s up to you, but IMHO searchResults sounds better... The question is do you even need it? Can you just filter places straight passing them down to render?